gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -843,6 +844,31 @@ func (vm *VMKeeper) QueryFile(ctx sdk.Context, filepath string) (res string, err
 	}
 }
 
+var rePkgPathAndSymbol = regexp.MustCompile(`^(.+)\.([a-zA-Z0-9_]+)$`)
```

Comment:
```suggestion
var rePkgPathAndSymbol = regexp.MustCompile(`^(.+)(:?\.([a-zA-Z0-9_]+))?$`)
```

Reply:
Hi @thehowl . I see what you're getting at. To match the input "gno.land/r/demo/boards.CreateRepost", we want the first group to be "gno.land/r/demo/boards" and the second group to be "CreateRepost" (or empty if the input doesn't have ".CreateRepost"). Instead, it just lets the second group be optional (empty) and puts the entire input into the first group. I'll try some other variations.

Reply:
As we discussed, now we just expect "package" (not "package.symbol"). So this code has been removed.